vllm/config/compilation.py

```diff
@@ -166,14 +166,21 @@ class CompilationConfig:
 
     # Top-level Compilation control
     level: Optional[int] = None
+    """
+    Level is deprecated and will be removed in the next release.
+    Please use mode. Currently all levels are mapped to mode.
+    """
+
+    # Top-level Compilation control
+    mode: Optional[int] = None
     """The level of compilation:
 
     - None: If None, we will select the default compilation level.
       For V1 engine this is 3, for V0 engine this is 0.
-    - 0: no compilation.
-    - 1: dynamo as is.
-    - 2: dynamo once.
-    - 3: piecewise compilation."""
+    - 0: NONE.
+    - 1: STOCK_TORCH_COMPILE.
+    - 2: DYNAMO_TRACE_ONCE.
+    - 3: VLLM_COMPILE, piecewise compilation."""
```

Comment:
Please add descriptions to these levels (cc @zou3519):

```suggestion
    - 0: NONE: no compilation
    - 1: STOCK_TORCH_COMPILE: just applying default torch.compile to model.
    - 2: DYNAMO_TRACE_ONCE: force a single Dynamo trace step by removing guards. Requires no dynamic-shape-dependent control-flow.
    - 3: VLLM_COMPILE, single Dynamo trace, custom vLLM caching, custom passes, custom cudagraph partitioning, etc."""
```